scrypt: the trait `From<base64ct::Error>` is not implemented for `HasherError`

I updated my dependencies today from scrypt 0.6.1 to 0.6.4 and my build fails now with the following error:

```
error[E0277]: `?` couldn't convert the error to `HasherError`
  --> /home/demurgos/.cargo/registry/src/github.com-1ecc6299db9ec823/scrypt-0.6.4/src/simple.rs:78:49
   |
78 |                 let pvec = Base64::decode_vec(p)?;
   |                                                 ^ the trait `From<base64ct::Error>` is not implemented for `HasherError`
   |
   = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
   = help: the following implementations were found:
             <HasherError as From<B64Error>>
             <HasherError as From<OutputError>>
             <HasherError as From<ParamsError>>
             <HasherError as From<base64ct::errors::InvalidLengthError>>
             <HasherError as From<password_hash::ParseError>>
   = note: required by `from`

error[E0277]: `?` couldn't convert the error to `HasherError`
  --> /home/demurgos/.cargo/registry/src/github.com-1ecc6299db9ec823/scrypt-0.6.4/src/simple.rs:85:49
   |
85 |                 let pvec = Base64::decode_vec(p)?;
   |                                                 ^ the trait `From<base64ct::Error>` is not implemented for `HasherError`
   |
   = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
   = help: the following implementations were found:
             <HasherError as From<B64Error>>
             <HasherError as From<OutputError>>
             <HasherError as From<ParamsError>>
             <HasherError as From<base64ct::errors::InvalidLengthError>>
             <HasherError as From<password_hash::ParseError>>
   = note: required by `from`
```

This may be caused by a mismatch between th `base64ct` versions:
- `scrypt` resolves `base64ct 1.0.0`
- `scrypt` resolves also `password-hash 0.1.1` which resolves `base64ct 0.2.0`

--- a/pbkdf2/Cargo.toml
+++ b/pbkdf2/Cargo.toml
@@ -17,7 +17,7 @@ crypto-mac = "0.10"
 rayon = { version = "1", optional = true }
 base64ct = { version = "1", default-features = false, optional = true }
 hmac = { version = "0.10", default-features = false, optional = true }
-password-hash = { version = "0.1", default-features = false, optional = true, features = ["rand_core"]  }
+password-hash = { version = "0.1.2", default-features = false, optional = true, features = ["rand_core"]  }
 sha1 = { version = "0.9", package = "sha-1", default-features = false, optional = true }
 sha2 = { version = "0.9", default-features = false, optional = true }
 

--- a/scrypt/Cargo.toml
+++ b/scrypt/Cargo.toml
@@ -14,7 +14,7 @@ readme = "README.md"
 [dependencies]
 base64ct = { version = "1", default-features = false, features = ["alloc"], optional = true }
 hmac = "0.10"
-password-hash = { version = "0.1", default-features = false, features = ["rand_core"], optional = true }
+password-hash = { version = "0.1.2", default-features = false, features = ["rand_core"], optional = true }
 pbkdf2 = { version = "0.7", default-features = false, path = "../pbkdf2" }
 salsa20 = { version = "0.7", default-features = false, features = ["expose-core"] }
 sha2 = { version = "0.9", default-features = false }

